oauth2client/contrib/django_util/__init__.py

```diff
@@ -162,7 +162,7 @@ def test_callback(sender, request, credentials, **kwargs):
 
    oauth2_authorized.connect(test_callback)
 
-"""
+"""  # noqa: E501
```

Comment:
This is for lines over 80 characters (the URLs).  I will add clarification when I fix this pr.